openlibrary/plugins/upstream/mybooks.py

```diff
@@ -112,7 +104,7 @@ def GET(self, username, key='want-to-read'):
 class readinglog_stats(delegate.page):
     path = "/people/([^/]+)/books/([a-zA-Z_-]+)/stats"
 
-    def GET(self, username, key='loans'):
+    def GET(self, username, key='want-to-read'):
```

Comment:
Hmm why did this change?

Reply:
As far as I could tell it was only being called with a key of "want-to-read", "already-read", or "currently-reading". I also removed the `loans` key completely, along with `KEYS` itself.

Reply:
Nevermind, I see now! You wrote it up 😊